frontend/src/app/uploads/DeviceLookup/DeviceLookup.test.tsx

```diff
@@ -14,6 +14,18 @@ import DeviceLookup from "./DeviceLookup";
 
 window.scrollTo = jest.fn();
 
+const supportedDiseaseTestPerformed = {
```

Comment:
**suggestion (non-blocking)**: could we clarify this object is for testing the duplicate supportedDisease.name by renaming it something like `duplicateNameSupportedDiseaseTestPerformed` or `anotherCovidSupportedDiseaseTestPerformed`?

Reply:
You bet! See [b11f2d4](https://github.com/CDCgov/prime-simplereport/pull/5834/commits/b11f2d41839ab18d772c09b3b73045619499766b)